integration/airflow/tests/integration/test_integration.py

```diff
@@ -105,99 +174,94 @@ def get_events(job_name: str = None):
     params = {}
 
     if job_name:
-        params = {
-            "job_name": job_name
-        }
+        params = {"job_name": job_name}
 
     # Service in ./server captures requests and serves them
+    backend_host = os.environ.get("BACKEND_HOST", "backend")
     r = requests.get(
-        'http://backend:5000/api/v1/lineage',
-        params=params,
-        timeout=5
+        f"http://{backend_host}:5000/api/v1/lineage", params=params, timeout=5
     )
     r.raise_for_status()
     received_requests = r.json()
-    log.info(f"Requests: {received_requests}")
     return received_requests
 
 
-@retry(
-    wait_fixed=2000,
-    stop_max_delay=15000
-)
+@retry(wait_fixed=2000, stop_max_delay=15000)
 def setup_db():
-    global airflow_db_conn
+    host = os.environ.get("AIRFLOW_DB_HOST", "postgres")
     airflow_db_conn = psycopg2.connect(
-        host="postgres",
-        database="airflow",
-        user="airflow",
-        password="airflow"
+        host=host, database="airflow", user="airflow", password="airflow"
     )
     airflow_db_conn.autocommit = True
-
-
-def test_integration(dag_id, request_path):
-    log.info(f"Checking dag {dag_id}")
-    # (1) Wait for DAG to complete
-    result = wait_for_dag(dag_id)
-    if not result:
-        sys.exit(1)
-
-    # (2) Read expected events
-    with open(request_path, 'r') as f:
+    return airflow_db_conn
+
+
+@pytest.fixture(scope="module", autouse=True)
+def airflow_db_conn():
+    yield setup_db()
+
+
+# @pytest.mark.parametrize("dag_id, request_path", params)
+# def test_integration(dag_id, request_path, airflow_db_conn):
+#     log.info(f"Checking dag {dag_id}")
+#     # (1) Wait for DAG to complete
+#     result = wait_for_dag(dag_id, airflow_db_conn)
+#     assert result is True
+
+#     # (2) Read expected events
+#     with open(request_path, "r") as f:
+#         expected_events = json.load(f)
+
+#     # (3) Get actual events
+#     actual_events = get_events()
+
+#     # (3) Verify events emitted
+#     log.info(f"failed to compare events for dag {dag_id}!")
+#     assert check_matches(expected_events, actual_events) is True
+#     log.info(f"Events for dag {dag_id} verified!")
+
+
+# @pytest.mark.parametrize("dag_id, request_dir", [("event_order", "requests/order")])
+# def test_integration_ordered(dag_id, request_dir: str, airflow_db_conn):
```

Comment:
What's up with those ordered tests?

Reply:
Oops, that's a mistake when testing new integration test in circleCI. Fixing it.